demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -26,7 +26,7 @@
 class IntegrationScript(ContentItem):
     type: str
     subtype: Optional[str]
-    docker_image: Optional[str]
+    docker_image: DockerImage = DockerImage("")
```

Comment:
why not Optional[DockerImage]?

Reply:
we could, but it can be annoying with mypy, we would need to do:
```
if script.docker_image: (to make sure its not `None`)
  ....
```